notifikasjon/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/river/ForespoerselLagretRiver.kt

```diff
@@ -66,18 +66,22 @@ class ForespoerselLagretRiver(
         context.publish(
             Key.EVENT_NAME to EventName.SAK_OPPRETT_REQUESTED.toJson(),
             Key.UUID to UUID.randomUUID().toJson(),
-            Key.DATA to "".toJson(),
-            Key.FORESPOERSEL_ID to forespoerselId.toJson(),
-            Key.ORGNRUNDERENHET to orgnr.toJson(),
-            Key.IDENTITETSNUMMER to fnr.toJson(),
+            Key.DATA to
+                mapOf(
+                    Key.FORESPOERSEL_ID to forespoerselId.toJson(),
+                    Key.ORGNRUNDERENHET to orgnr.toJson(),
+                    Key.IDENTITETSNUMMER to fnr.toJson(),
+                ).toJson(),
         )
 
         context.publish(
             Key.EVENT_NAME to EventName.OPPGAVE_OPPRETT_REQUESTED.toJson(),
             Key.UUID to UUID.randomUUID().toJson(),
```

Comment:
Kan vi fjerne UUID herfra siden vi har fjernet den fra opprettOppgaveService sine startkeys? Eller må vi gjøre det i et senere steg for å unngå kluss med meldinger som er underveis?

Reply:
Vi vil ikke fjerne `Key.UUID`. Det fjernes fra `startKeys` i oppgave-servicen fordi den antar da at nøkkelen ligger nested inni en `Key.DATA`, som vi ikke gjør.

Reply:
Skjønner 👍 Takk for oppklaring!